spec/inputs/pipe_spec.rb

```diff
@@ -1,4 +1,5 @@
 # encoding: utf-8
+return if LogStash::Environment.windows?
```

Comment:
Would it be better to tag the suite with `unix` and exclude it when running the test? We could use the same behavior for windows only plugins.